app/models/manageiq/providers/amazon/refresh_helper_methods.rb

```diff
@@ -41,6 +41,17 @@ def add_instance_disk(disks, size, name, location)
     super(disks, size, name, location, "amazon")
   end
 
+  def add_block_device_disk(disks, name, location)
+    disk = {
+      :device_name     => name,
+      :device_type     => "disk",
+      :controller_type => "amazon",
+      :location        => location,
+    }
+    disks << disk
```

Comment:
@gberginc - is it possible to get any more information about the block devices, maybe size and free space? 
Can device_type be any string? If so, can we distinguish between instance store volume or EBS volume here?